docs/Projects/TAD/ADRs/0004-software-stack.md

```diff
@@ -0,0 +1,78 @@
+# ADR-0004 Software Stack for TAD
+
+## Context
+
+For building our own TAD solution, we need to choose a software stack. During our earlier
+POCs and market research, we gathered insight and information on technologies to use
+and which not to use.
+
+During further discussions and brainstorm sessions, a software stack was chosen
+that accommodates our needs best.
+
+While more fine grained [requirements](../Existing-Tools/Comparison/requirements) are listed elsewhere,
+some key requirements are:
+
+* the tool is locally runable with docker
+* the tool is runnable as user as a local Docker solution
+* the tool is runnable on a cloud platform based on Kubernetes
+* the tool supports multiple organizations, teams and projects
+* the tool supports Oauth2
+
+## Assumptions
+
+We stick to suitable programming languages. As most AI related
+tooling is written in Python, this language is the logical choice for our development as well.
+
+Currenlty we do not see the need for a separate web GUI framework.
+it is preferred to bundle backend and frontend in one solution.
+
+As part of a Dutch government organization, we need to adhere to all dutch laws and standards, like
+[Digitoegankelijk](https://www.digitoegankelijk.nl/) and
+[WCAG Guidelines](https://wcag.nl/kennis/richtlijnen/wcag-2-1-richtlijnen/) .
+
+We do not forsee high load or many static files, therefore, a separate webserver like
+nginx, apache is not required. Static files will be served directly by the backend.
```

Comment:
I do not think static files will be served by the backend. Only the HTML files that include Jinja2. I will not serve images through the backend

Reply:
Where will images/static files come from then and do we need to include that?

Reply:
For fastAPI I tried this (which is why I mentioned it would be served by the backend):
`
app.mount("/static",` StaticFiles(directory="static"), name="static")
`
as mentioned here https://fastapi.tiangolo.com/tutorial/static-files/

Reply:
That is indeed fine